<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8BL3MIG" failures="1" tests="7" name="Test" time="85.913" errors="0" timestamp="2024-12-18T16:10:01 PKT">
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_account_registration" time="10.294"/>
  <testcase classname="testCases.TC002_LoginTest" name="verify_login" time="2.438"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDDT" time="11.678">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDataDrivenTest.verify_loginDDT(TC003_LoginDataDrivenTest.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDDT" time="11.457"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDDT" time="11.5"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDDT" time="11.773"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDDT" time="3.281"/>
</testsuite> <!-- Test -->
